lib/devise/encryptable/encryptors/pbkdf2.rb

```diff
@@ -0,0 +1,14 @@
+module Devise
+  module Encryptable
+    module Encryptors
+      class Pbkdf2 < Base
+        def self.digest(password, stretches, salt, pepper)
+          digest = OpenSSL::Digest::SHA512.new
+          len = digest.digest_length
+          iterations = 2**stretches
```

Comment:
`stretches` is part of the `devise` config API. 

I agree; it's not a good 1:1 comparison when doing pbkdf2 vs bcrypt. I was aiming here for something approximate. You'd want `devise.config.stretches = 14` to get something closer to adequate for pbkdf2.